pylint/lint.py

```diff
@@ -54,6 +46,12 @@
 
 
 MANAGER = astroid.MANAGER
+USAGE = __doc__ + """
+Argument:
```

Comment:
This shouldn't be documented. From my point of view, this is the hack we're supporting for linter-atom's sake.